handbook/company/pricing-features-table.yml

```diff
@@ -743,39 +727,15 @@
   productCategories: [Device management]
   pricingTableCategories: [Device management]
   moreInfoUrl: https://github.com/fleetdm/fleet/issues/18865
-- industryName: Self-service portal for users to install apps
-  description: Add optional apps for your end users through Fleet Desktop (for macOS, Windows, and Linux).
+- industryName: Self service application installation
+  description: Allow end users to install apps through Fleet Desktop for macOS, Linux, and Windows.
   tier: Premium
   jamfProHasFeature: yes
   jamfProtectHasFeature: no
   usualDepartment: IT
   productCategories: [Device management]
   pricingTableCategories: [Device management]
   moreInfoUrl: https://github.com/fleetdm/fleet/issues/17587
-- industryName: Install Apple App Store apps through Volume Purchasing Program (VPP).
-  description: Offer licenses for Photoshop and other App Sore apps for your end users.
-  tier: Premium
-  jamfProHasFeature: yes
-  jamfProtectHasFeature: no
-  comingSoonOn: 2024-08-11
-  waysToUse:
-    - description: macOS coming soon (2024-07-15). #customer-rosner
```

Comment:
Hey @mike-j-thomas we're using this features.yml to track customer promises: features we committed to in an order form.

To track the promises, we add a customer codename for tracking purposes.

Do you think we can keep the `waysToUse` section for this feature so that we can continue tracking the `#customer-rosner` commit?